ui/src/components/overview/MobileClientBuildHistoryList.scss

```diff
@@ -0,0 +1,48 @@
+.build-history-item, .build-history-item-top {
+  padding-top: 1em;
+  padding-bottom: 1em;
+}
+
+.build-history-item-top {
+  border: 1px solid grey;
```

Comment:
This grey appears to be a lot darker then the grey we have elsewhere, can we align these.